buildkite/src/Jobs/Test/ReplayerTest.dhall

```diff
@@ -0,0 +1,23 @@
+let S = ../../Lib/SelectFiles.dhall
+
+let Pipeline = ../../Pipeline/Dsl.dhall
+
+let JobSpec = ../../Pipeline/JobSpec.dhall
+
+let ReplayerTest = ../../Command/ReplayerTest.dhall
+
+let dependsOn =
+      [ { name = "ArchiveNodeArtifact", key = "build-archive-deb-pkg" } ]
+
+in  Pipeline.build
+      Pipeline.Config::{
+      , spec = JobSpec::{
+        , dirtyWhen =
+          [ S.strictlyStart (S.contains "src")
```

Comment:
Is there a more restrictive dirtywhen spec that would make sense here?

Reply:
A more restrictive dirtyWhen could be src/lib , which is only a slight difference ... we want to be sure anything that might affect processing of transactions gets caught

But maybe we really want `src/`, because the archive lib code is in `src/app/`.